k8s/resources/study/dynamic-simulation-server-deployment.yaml

```diff
@@ -26,6 +26,19 @@ spec:
               name: dynamic-simulation-server-configmap-specific-volume
             - mountPath: /home/powsybl/.itools
               name: dynawo-itools-configmap-volume
+          env:
```

Comment:
Should be put in common

Reply:
DONE! moved to patch deployments-springboot-s3-config.yaml